exercises/queen-attack/description.md

```diff
@@ -9,17 +9,41 @@ A chessboard can be represented by an 8 by 8 array.
 So if you are told the white queen is at `c5` (zero-indexed at column 2, row 3) and the black queen at `f2` (zero-indexed at column 5, row 6), then you know that the set-up is like so:
 
 ```text
-  a b c d e f g h
-8 _ _ _ _ _ _ _ _ 8
-7 _ _ _ _ _ _ _ _ 7
-6 _ _ _ _ _ _ _ _ 6
-5 _ _ W _ _ _ _ _ 5
-4 _ _ _ _ _ _ _ _ 4
-3 _ _ _ _ _ _ _ _ 3
-2 _ _ _ _ _ B _ _ 2
-1 _ _ _ _ _ _ _ _ 1
-  a b c d e f g h
+  a b c d e f g h       a b c d e f g h       a b c d e f g h
+8 _ _ _ _ _ _ _ _ 8   8 _ _ _ _ _ _ _ _ 1   8 _ _ _ _ _ _ _ _ 8
+7 _ _ _ _ _ _ _ _ 7   7 _ _ _ _ _ _ _ _ 2   7 _ _ _ W _ _ _ _ 7
+6 _ _ _ W _ _ _ _ 6   6 _ B _ _ _ W _ _ 3   6 _ _ _ _ _ _ _ _ 6
+5 _ _ _ _ _ _ _ _ 5   5 _ _ _ _ _ _ _ _ 4   5 _ _ _ _ _ _ _ _ 5
+4 _ _ _ _ _ _ _ _ 4   4 _ _ _ _ _ _ _ _ 5   4 _ _ _ _ _ _ _ _ 4
+3 _ _ _ _ _ _ B _ 3   3 _ _ _ _ _ _ _ _ 6   3 _ _ _ _ _ _ _ _ 3
+2 _ _ _ _ _ _ _ _ 2   2 _ _ _ _ _ _ _ _ 7   2 _ _ _ B _ _ _ _ 2
+1 _ _ _ _ _ _ _ _ 1   1 _ _ _ _ _ _ _ _ 8   1 _ _ _ _ _ _ _ _ 1
+  a b c d e f g h      a b c d e f g h        a b c d e f g h
```

Comment:
@T-O-R-U-S You have modified the first example of queens attacking so that it no longer matches the text before it. Not only the positions are different than in the text (the example shows d6 and g3, not c5 and f2 as the text says), but the text is clearly written for a single example, not for 3. Maybe we can just revert this part to the original and only leave the three new examples of queens not attacking?

Additionally, the CI is complaining about linting problems. Those need to be fixed before we can approve the PR.